app/models/user.rb

```diff
@@ -1,12 +1,24 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, authentication_keys: [:name]
```

Comment:
- [ ] Please add `:confirmable` at the end of line no. 5 because this requirement so kindly fulfil all the requirements:slightly_smiling_face: 

![Screenshot from 2024-01-11 23-18-46](https://github.com/Estete9/blog-app-ror/assets/45226848/6f28796e-c730-4017-9fcf-832aae7a179d)